SC4Cartographer/SC4Cartographer/MapApperance/MapApperanceSerializer.cs

```
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using System.Drawing;
+
+
+namespace SC4CartographerUI
+{
+    public class MapApperanceSerializer
+    {
+        private const string tempFilename = "map_appearance_autosave.sc4cart";
+        private readonly string tempFilePath = Path.Combine(Path.GetTempPath(), tempFilename);
+        private const int VERSION = 1;
+
+        public void SaveToUserTempFolder(MapCreationParameters parameters)
```

Like `TryLoadFromUserTempFolder`, `SaveToUserTempFolder` should silently continue when an exception occurs when we save to the temp directory. 

This is functionality that will run when the user closes the form, we do not want to interupt that with an error message.